cumulusci/cli/tests/test_cci.py

```diff
@@ -235,10 +233,8 @@ def test_project_init(self, click):
                 "_TEST",  # test_name_match
             )
             click.confirm.side_effect = (True, True)  # is managed?  # extending?
-            config = mock.Mock()
-            config.global_config.project__test__name_match = "_TEST"
 
-            run_click_command(cci.project_init, obj=config)
+            run_click_command(cci.project_init)
```

Comment:
I stopped passing a mock config object for the `cci project init` test so this should be a real integration test now.